js/editor.js

```diff
@@ -73,3 +75,37 @@ $(document).ready(function () {
 
   });
 });
+
+var Range = ace.require('ace/range').Range;
+var errors = {};
+var highlight_errors = function(rid, re) {
```

Comment:
Pass `editor_id` here
Rename `rid` to `error_list_id`
This is global again, no need to use `var`